### PR TITLE
2.3 Toasts and Notifications

### PR DESCRIPTION
**Pull request by priyanshu688**
In this file, I have Written all the information about Working on getting Toast and Notification in an Android App. The language used is Java. Briefly explain with the required steps  in order to create toast and notification.

## 2.3 Working on getting Toasts and Notifications📜

# ISSUENUM #645  @VikasPandey121 please review this.

### CODE REVIEW

**Comment by VikasPandey121** [comment]
Awsome work @priyanshu688 just minor changes requested:

- Make sub-titles if there are any
- Add pictures to make it an infographic and centre all the images

**Comment by VikasPandey121** [comment]
Add issue tag like this #645

**Comment by priyanshu688** [comment]
Change the issue tag...but how to add the pictures ..can you please tell me??

File: Android_Development_With_Java/2  Android UI Widgets/2.3 Toasts and Notifications [hunk 1]
```diff
@@ -0,0 +1,137 @@
+WORKING ON GETTING TOASTS & NOTIFICATIONS
+
+                                                          TOAST
+                                                          
+Toast is actually a simple feedback about an operation in a small pop-up that is displayed on the current activity UI Screen. It basically fills the amount of space required for the message and the current activity remains visible and interactive. Toasts automatically disappear after a given timeout. 
+Toast is basically used when the App is in the foreground and you want the users to take action then only we display toast to the users.
```
Comment (priyanshu688) [inline Android_Development_With_Java/2  Android UI Widgets/2.3 Toasts and Notifications:6]: image

File: Android_Development_With_Java/2  Android UI Widgets/2.3 Toasts and Notifications [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+                   Methods                                          Description
+             public void show()                           displays a toast message
+             public void setMargin(float                  changes the horizontal and vertical differences.
+          horizontalMargin,float verticalMargin)         
+
+◆ Let's take a glance of how to create an Android App to show a Toast Message :-
+
+  Step 1 - Firstly We need to create an XML File and a Java File. In XML file, we basically wrote the code of the UI, how our file looks like designing and all. But In Java File, we wrote the code for the implementation.
+
```
Comment (priyanshu688) [inline Android_Development_With_Java/2  Android UI Widgets/2.3 Toasts and Notifications:33]: image

Comment (priyanshu688) [inline Android_Development_With_Java/2  Android UI Widgets/2.3 Toasts and Notifications:33]: image

File: Android_Development_With_Java/2  Android UI Widgets/2.3 Toasts and Notifications [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+       Now display the created Toast message using the show() method:-  public void show ()
+      Toast.makeText(getApplicationContext(), "This a toast message",Toast.LENGTH_LONG).show();       
+                  
+           
+                                         NOTIFICATIONS
+                                         
+Notification basically provides us the short, timely information about the events in your app while it's not in use. A Notification in its basic and the compact, which is also known as the "Collapsed form" displays an icon, a title and the single line text related to it.
+Whenever we create the projects with Android Studio includes the necessary dependencies to use NotificationCompat. Here you need to verify that your module level build.gradle file includes the following dependencies.
+
```
Comment (priyanshu688) [inline Android_Development_With_Java/2  Android UI Widgets/2.3 Toasts and Notifications:78]: image

**Comment by VikasPandey121** [comment]
You can use `Markdown Monster icon` Please look up for markup language to know more about it.

**Pull request closed without merging**